omegaconf/omegaconf.py

```diff
@@ -568,6 +569,24 @@ def update(cfg: Container, key: str, value: Any = None) -> None:
             idx = int(last)
             root[idx] = value
 
+    @staticmethod
+    def to_yaml(cfg: Container, resolve: bool = False, sort_keys: bool = False) -> str:
```

Comment:
1. I think this is a good opportunity to change the default of resolve to True. (but be sure to mention in in the deprecation warning and the docs).

2. Force named parameters for everything after cfg, see [this](https://stackoverflow.com/questions/2965271/forced-naming-of-parameters-in-python). This will ensure that it's easier to extend this function in the future without worrying about people calling the flags positionally.